### DISCUSSION TITLE
出现了JSON解析错误

### DISCUSSION THREAD

**Discussion by G1iderK** [GaiZhenbiao/ChuanhuChatGPT#560] [Q&A]
按照教程配置完成后，登录发送信息，返回：

JSON解析错误。请重置对话。收到的内容: “error”: { “message”: “You exceeded your current quota, please check your plan and billing details.”, “type”: “insufficient_quota”, “param”: null, “code”: null }}

想问一下这个是要怎么解决哇？是因为没有买plus么？

**Comment by GaiZhenbiao** [comment]
你的免费余额用完了

**Comment by flyingSpagetiMonstr** [comment]
我也遇到了一样的问题，貌似是送的余额过期了
Grant 1	$18.00	Expired 2023年4月1日
TM我还一点没用啊
